docs/flashbots-auction/searchers/advanced/rpc-endpoint.mdx

```diff
@@ -119,6 +119,86 @@ example response:
 }
 ```
 
+### eth_sendPrivateTransaction
+
+`eth_sendPrivateTransaction` is used to send a single transaction to Flashbots. Flashbots will attempt to send the transaction to miners for the next 25 blocks. See [Private Transactions](/flashbots-auction/searchers/advanced/private-transaction.mdx) for more info.
+
+This method has the following JSON-RPC format:
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "eth_sendPrivateTransaction",
+  "params": [{
+    tx,               // String, raw signed transaction
+    maxBlockNumber,   // Number, optional. Highest block number in which the transaction should be included.
+  }]
+}
+```
+
+example:
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "eth_sendPrivateTransaction",
+  "params": [{
+    "tx": "0x123abc...",
+    "maxBlockNumber": 14322777,
```

Comment:
As per above comment(Chris), maxBlockNumber not an `integer` it should be a string.
```suggestion
    "maxBlockNumber": "0xcd23a0",
```